### PR TITLE
Copy ibis feedstock recipe

### LINKED ISSUE #2523: Update conda-forge recipe from released version
We introduced some changes to the recipe to release 1.4. We need to copy the recipe to this repo (`ci/recipe/meta.yaml`), so the packaging build uses those changes.

### PR DESCRIPTION
**Pull request by matthewmturner**
closes #2523 

@datapythonista

### CODE REVIEW

File: ci/recipe/meta.yaml [hunk 1]
```diff
@@ -8,18 +8,20 @@
 # - Set the `number` in the `build` section to the appropriate build number
 # - Remove this comment from the beginning of the file
 
+{% set version = "1.4.0" %}
+
```
Comment (datapythonista) [inline ci/recipe/meta.yaml:12]: Can you remove this? We edit this file via bash for the packaging build, and I think we add this automatically.

File: ci/recipe/meta.yaml [hunk 2]
```diff
@@ -8,18 +8,20 @@
...
 package:
   name: ibis-framework
   version: {{ version }}
 
 source:
   url: https://github.com/ibis-project/ibis/archive/{{ version }}.tar.gz
+  sha256: dd3e3d6d495d4ed4f7e5b8ec523949bc54f6fc89f31ca60ef3dc103de8d37635
```
Comment (datapythonista) [inline ci/recipe/meta.yaml:19]: Same here.

**Review by datapythonista — COMMENTED** [review]
Looks good, just two things that will make the CI and need to be removed, but other than that looks good.

**Review by datapythonista — APPROVED** [review]
Great, thanks for taking care of this @matthewmturner

File: ci/recipe/meta.yaml [hunk 3]
```diff
@@ -86,9 +85,6 @@ about:
...
 extra:
   recipe-maintainers:
-    - cpcloud
-    - mariusvniekerk
-    - wesm
-    - kszucs
-    - xmnlab
     - jreback
+    - xmnlab
+    - datapythonista
```
Comment (jreback) [inline ci/recipe/meta.yaml:90]: blank line at the end

**Review by jreback — CHANGES_REQUESTED** [review]
tiny comment, let's see this build correctly (docs build), though unrelated i think.

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @matthewmturner

**Pull request merged**